docs/source/models/intervention_models/alzheimers/hypothetical_treatment.rst

```diff
@@ -30,4 +30,234 @@
 
 ========================================
 Hypothetical Alzheimer's Treatment
-========================================
\ No newline at end of file
+========================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - BBBM
+    - Blood-based biomarker
+    - BBBM tests measure blood plasma protein levels and are less invasive than CSF (Cerebrospinal Fluid) or PET (Positron Emission Topography) tests
+  * - MCI
+    - Mild cognitive impairment
+    -
+
+Intervention Overview
+-----------------------
+
+The hypothetical treatment intervention is triggered by a positive BBBM test, and has the effect slowing the progression
+from pre-clinical to MCI state. This progression rate is set by a :ref:`time-dependent hazard function <2021_cause_alzheimers_presymptomatic_mci_transition_data_table>`
+which is multiplied by a hazard ratio H < 1 when a simulant has an active treatment effect in order to slow the progression.
+This effect can wane over time (udpated each time step) and when the effect fully expires, H returns to 1. 
+
+This treatment is hypothetical and we don't have confirmed information about the mechanism.
+
+
+.. list-table:: Affected Outcomes
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Modeled?
+    - Note (ex: is this relationship direct or mediated?)
+  * - Hypothetical treatment effect
+    - Slows progression to MCI (time-dependent effect size)
```

Comment:
The affected outcome should be the hazard rate for the transition from BBBM to MCI, and the effect should be multiplying the hazard rate by the hazard ratio. See these [docs for treatment of neonatal sepsis](https://vivarium-research.readthedocs.io/en/latest/models/intervention_models/neonatal/antibiotics_intervention.html) for an example.